decidim-admin/app/assets/javascripts/decidim/admin/newsletters.js.es6

```diff
@@ -66,5 +66,14 @@ $(() => {
         }
       });
     })
+
+    $(".form.newsletter_deliver").on("change", function() {
```

Comment:
here `$form.on("change", function() {` is the same as `$(".form.newsletter_deliver").on("change", function() {`